src/5.1.thats_the_way.py

```
@@ -0,0 +1,37 @@
+"""
+Created on Fri Apr  4 17:10:05 2025
+
+@author: USER
+"""
+import os
+
+def thats_the_way(folder_path):
+    """Returns a list of all filenames in the given directory that start with the prefix 'deep'.
+
+    Parameters:
+    folder_path (str): The path to the directory to search in.
+
+    Returns:
+    returned_files[str]: A list of filenames (not full paths) that start with 'deep'.
+
+    Notes:
+    - Only files (not subdirectories) are considered.
+    """
+
+    if not os.path.isdir(folder_path):
+        return []
```

Return error instead of print and return none -1